components/tooltip/tooltip.js

```diff
@@ -148,6 +148,11 @@ class Tooltip extends RtlMixin(LitElement) {
 			 * @type {number}
 			 */
 			offset: { type: Number }, /* tooltipOffset */
+			/**
+			 * ADVANCED: Only show the tooltip if we detect the target element is truncated
+			 * @type {boolean}
+			 */
+			onlyShowIfTruncating: { type: Boolean, attribute: 'only-show-if-truncating' },
```

Comment:
Nit: I might consider renaming this `showTruncatedOnly`.  I'd slightly less wordy, and there might be a tiny bit of precedent for it.  None of the names I can think of are amazing tbh.  🤷‍♂️